client/src/pages/OfferHelp.js

```diff
@@ -65,6 +66,12 @@ const Step1 = (props) => {
           {Object.entries(answers).map(([answer, checked], i) => (
             <WizardCheckboxItem
               key={i}
+              id={
+                GTM.offerHelp.volunteer +
+                props.currentStep +
+                GTM.offerHelp.orgbtn +
+                GTM.offerHelp.donor
+              }
```

Comment:
I don't think this will work. You are setting all the ids to each element... @jyclam What do you think? This is creating the wrong tag. 

![image](https://user-images.githubusercontent.com/25408660/86729056-092f1b00-bffb-11ea-93b1-740746a9a4a0.png)

Reply:
These are the checkboxes

Reply:
Ill work on a fix for this one.